src/app/components/question-editor/question-editor.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { QuestionService } from 'src/app/service/question.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { QuestionToUpdateModel } from 'src/app/models/question-to-update-model';
+import { QuestionModel } from 'src/app/models/question-model';
+
+@Component({
+  selector: 'app-question-editor',
+  templateUrl: './question-editor.component.html',
+  styleUrls: ['./question-editor.component.scss']
+})
+export class QuestionEditorComponent implements OnInit {
+
+  formData: QuestionModel;
+  isCreating = false;
+  submitButtonText = "Zaktualizuj pytanie";
```

Comment:
musi to byc zadeklarowane tutaj ?

Reply:
zostawialem form data bo on przesyla do kompoentu dane i isCreating bo ta flaga steruje wygladam forlumarza w zależnosci od tego czy tworzymy pytania czy je edytujemy